application/src/main/java/org/opentripplanner/updater/trip/gtfs/updater/mqtt/MqttGtfsRealtimeUpdater.java

```diff
@@ -19,6 +19,10 @@
 import org.opentripplanner.updater.spi.GraphUpdater;
 import org.opentripplanner.updater.spi.UpdateResult;
 import org.opentripplanner.updater.spi.WriteToGraphCallback;
+import org.opentripplanner.updater.trip.BackwardsDelayPropagationType;
+import org.opentripplanner.updater.trip.UpdateIncrementality;
```

Comment:
Should these two be inside the gtfs package as I don't think they are used by siri?

Reply:
No, it's also used by SIRI.